scripts/spack_packages/packages/care/package.py

```diff
@@ -37,22 +37,31 @@ class Care(CachedCMakePackage, CudaPackage, ROCmPackage):
     variant('implicit_conversions', default=False, description='Enable implicit conversions to/from raw pointers')
     variant('loop_fuser', default=False, description='Enable loop fusion capability')
 
-    depends_on('cmake@3.14.5:', type='build')
-    depends_on('cmake@3.21:', when='+rocm', type='build')
+    depends_on('cmake@3.18:', type='build', when='@develop')
+    depends_on('cmake@3.21:', type='build', when='+rocm')
+    depends_on('cmake@3.14:', type='build', when='@0.10.0:')
```

Comment:
Maybe a small thing, but if possible, please order in descending version order like the BLT lines below.  This just helps with my understanding.

Reply:
Fixed.